tests/testsrc/test262.properties

```diff
@@ -1150,17 +1531,13 @@ built-ins/Math 11/327 (3.36%)
 
 built-ins/NaN 0/6 (0.0%)
 
-built-ins/NativeErrors 36/139 (25.9%)
-    AggregateError/newtarget-proto-fallback.js
-    AggregateError/proto-from-ctor-realm.js
```

Comment:
I assume, that these tests are now removed, because `AggregateError` is an unsupported feature?